packages/compose/README.md

```diff
@@ -86,6 +86,41 @@ _Returns_
 
 -   Component class with generated display name assigned.
 
+### debounce
+
+A simplified and properly typed version of lodash's `debounce`, that
+always uses timers instead of sometimes using rAF.
+
+Creates a debounced function that delays invoking `func` until after `wait`
+milliseconds have elapsed since the last time the debounced function was
+invoked, or until the next browser frame is drawn. The debounced function
```

Comment:
Looks like `, or until the next browser frame is drawn` unintentionally survived the `rAF` purge. We should probably remove it, as there's nothing in this function that is related to the browser rendering pipeline now.